### PR TITLE
Order summary

### PR DESCRIPTION
**Pull request by eliseydah**
Add create invoice

### CODE REVIEW

File: frontend/src/views/InvoiceView.vue [hunk 1]
```diff
@@ -109,128 +133,104 @@ onMounted(async () => {
...
-      </div>
-      <div
-        class="flex flex-row justify-between items-center"
-        v-for="(item, idx) in serviceDates"
-        :key="item.id"
-      >
+    </div>
+    <OrderSummary :order-info="orderInfo" />
+    <Card class="my-2">
+      <template #content>
```
Comment (fr-ser) [inline frontend/src/views/InvoiceView.vue:149]: A lot of space is used here on the right side of those small inputs.
Why not use something that uses the full width and can wrap around?

image

File: frontend/src/components/orders/OrderSummary.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          :to="`${ROUTES.CLIENT.path}/${props.orderInfo.client_id}/edit`"
+        >
+          {{ props.orderInfo.client.first_name }} {{ props.orderInfo.client.last_name }}
+          <i class="pi pi-external-link ml-1"></i>
+        </router-link>
+      </div>
+      <div>
+        <span class="font-bold">Skonto: </span>
+        <span>{{ props.orderInfo.can_have_cash_discount ? "Ja" : "Nein" }}</span>
+      </div>
```
Comment (fr-ser) [inline frontend/src/components/orders/OrderSummary.vue:34]: What about discount_duration and discount_percentage?

File: frontend/src/components/orders/OrderSummary.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import type { Order } from "@/global/types/entities";
+import { ROUTES } from "@/router";
+
+const props = defineProps<{
+  orderInfo: Order;
+}>();
+</script>
+<template>
+  <Card class="my-2">
+    <template #content>
```
Comment (fr-ser) [inline frontend/src/components/orders/OrderSummary.vue:13]: Maybe here we can also think about a design that uses the full width of the screen?

File: frontend/src/compositions/useNotifications.ts [hunk 4]
```diff
@@ -75,6 +75,13 @@ export default function useNotifications() {
       life: 3000,
     });
   };
+  const showCreateInvoiceNotification = () => {
+    toast.add({
+      severity: "info",
+      summary: "New invoice was created",
```
Comment (fr-ser) [inline frontend/src/compositions/useNotifications.ts:81]:
```suggestion
      summary: "Eine neue Rechung wurde erstellt",
```

File: frontend/src/views/InvoiceView.vue [hunk 5]
```diff
@@ -87,6 +93,24 @@ const allItemsSum = computed(() => {
...
+    items: invoiceItemsArray.value,
+    service_dates: serviceDates.value
+      .filter(function (item) {
+        if (item.date) {
+          return true;
+        }
+      })
+      .map(function (element) {
+        return formatDateToIsoString(element.date as Date);
+      }),
```
Comment (fr-ser) [inline frontend/src/views/InvoiceView.vue:108]:
```suggestion
    service_dates: serviceDates.value.filter(item => Boolean(item.date))
      .map(element => formatDateToIsoString(element.date as Date)),
```

**Comment by fr-ser** [comment]
Are all branches active or should some be deleted?

image

File: frontend/src/components/orders/OrderSummary.vue (hunk 3 above)
Comment (eliseydah) [reply frontend/src/components/orders/OrderSummary.vue:13]: image

File: frontend/src/views/InvoiceView.vue (hunk 1 above)
Comment (eliseydah) [reply frontend/src/views/InvoiceView.vue:149]: image

File: frontend/src/components/orders/OrderSummary.vue (hunk 2 above)
Comment (eliseydah) [reply frontend/src/components/orders/OrderSummary.vue:34]: image

**Pull request merged**